client/src/components/Q&A/AnswerRow.jsx

```diff
@@ -1,9 +1,8 @@
-/* eslint-disable import/extensions */
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
+import Report from './Report';
 
-
-export default function AnswerRow({ questionId }) {
+export default function AnswerRow({ questionId, incrementCount }) {
```

Comment:
take out incrementCount?

Reply:
You need a separate counter for # of answers.

Reply:
Nice catch! Removed as well